control-plane/csi-driver/src/bin/node/filesystem_ops.rs

```diff
@@ -290,6 +294,103 @@ impl FileSystemOps for XFs {
     }
 }
 
+#[async_trait]
+impl FileSystemOps for BtrFs {
+    async fn create(&self, device: &str) -> Result<(), Error> {
+        let binary = format!("mkfs.{}", "btrfs");
+        let output = Command::new(&binary)
+            .arg(device)
+            .output()
+            .await
+            .map_err(|error| format!("failed to execute {binary}: {error}"))?;
+        ack_command_output(output, binary)
+    }
+
+    fn mount_flags(&self, mount_flags: Vec<String>) -> Vec<String> {
+        mount_flags
+    }
+
+    fn unmount_on_fs_id_diff(
+        &self,
+        device_path: &str,
+        fs_staging_path: &str,
+        volume_uuid: &Uuid,
+    ) -> Result<(), Error> {
+        mount::unmount_on_fs_id_diff(device_path, fs_staging_path, volume_uuid)
+    }
+
+    /// Btrfs filesystem needs an check.
```

Comment:
What does needs a check mean?

Reply:
This check as per the docs is suggested before the metadata change, in read-only mode, this option exists to calm potential panic when users are going to run the checker.

Maybe the comment is not correct?

Reply:
Yeah just looks a bit out of place to have a blanket statement like that on top of the function, I'd remove it or elaborate more on it in regards to repair.

Reply:
I don't think it does any repair(dangerous operation as termed by btrfs) it's just to allow us the uuid change. Should I remove the comment then?

Reply:
Yeah or perhaps document that we don't do any repair then as it's dangerous and why.

Reply:
Done.